Sources/Domain/Update/DialogTexts.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  DialogTexts.swift
+//  Uppsala
+//
+//  Created by Suita Fujino on 2019/01/05.
+//  Copyright © 2019 Suita Fujino. All rights reserved.
+//
+
+/**
+ A data class to hold the messages for the alert dialog.
+ */
+public struct DialogTexts {
+    
+    /// The title of the alert dialog.
+    public let title: String
+    /// The message of the alert dialog.
+    public let message: String
+    
+    /**
+     Initializer.
+     
+     - Parameters:
+       - title: The title of the dialog. `nil` will be replaced with the default text.
+       - message: The message of the dialog. `nil` will be replaced with the default text.
+     */
+    public init(title: String?, message: String?) {
+        // TODO: Localize
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Scior/Uppsala/blob/3e3bf287e532edf7eff7664ca9a2a4565256aa13/Sources/Domain/Update/DialogTexts.swift#L27"></span>TODOs should be resolved (Localize).</td>
    </tr>
  </tbody>
</table>